managed/models/database.go

```diff
@@ -893,7 +900,7 @@ func SetupDB(sqlDB *sql.DB, params *SetupDBParams) (*reform.DB, error) {
 			params.Logf("Creating database %s and role %s", databaseName, roleName)
 		}
 		// we use empty password/db and postgres user for creating database
-		db, err := OpenDB(params.Address, "", "postgres", "")
+		db, err := OpenDB(params.Address, "", "postgres", "", "", "")
```

Comment:
We shall use SSL here if it was provided by the user.

Reply:
We use this place only for our internal needs. This place essentially does the following:
1. If a user with such parameters and a database has not been created, we create a user and a database under root rights.
2. We give the user rights to this database and subsequently use this user for interaction. I'm not sure if it would be a good idea to ask the client for root permissions in order to create the database and user we need. I suppose if the client does it himself and gives the rights necessary in his opinion, it will be better.

Reply:
Yes, let's do this please. With external databases we cannot expect to have `postgres` user with no password available.

Reply:
error message added